packages/v4/src/content/design-guidelines/demos/card-view/card-view.md

```diff
@@ -0,0 +1,69 @@
+---
+id: Card view
+section: demos
+---
+
+
+A **card view** is a grid of cards in a gallery to facilitate browsing. Card views are typically used to present data set summaries, allowing users to drill down into any card to see more detailed content.
+
+<img src="./img/card-view.png" alt="card view" />
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
```

Comment:
Re-commenting to make sure Alana's previous suggestion doesn't get lost since it doesn't seem to have been addressed.

Two things, change "Do NOT" to "Never" and hyperlink "table" and "list view"!

![image](https://user-images.githubusercontent.com/51165119/117982528-1516f800-b304-11eb-9d5e-89f4a3b89bb7.png)